pages/dkp/kommander/2.2/workspaces/applications/catalog-applications/custom-applications/add-create-git-repo/index.md

```diff
@@ -0,0 +1,72 @@
+---
+layout: layout.pug
+navigationTitle: Create a Git Repository
+title: Create a Git Repository
+menuWeight: 10
+excerpt: Create a Git Repository in the Project namespace
```

Comment:
```suggestion
excerpt: Create a Git Repository in the Workspace namespace
```

Please adapt all these pages to Workspace context

Reply:
^^ bump on this - these pages were copied over from projects/, need to update all the project references to Workspace context